tests/datatypes/test_str.py

```diff
@@ -574,6 +574,18 @@ def test_rjust(self):
                 print(err)
             """)
 
+    def test_rpartition(self):
+        self.assertCodeExecution("""
+            Stri = "Hello World!"
```

Comment:
flake8 won't catch the problems with test code, but your test code should be flake8 compliant, too. Variable names shouldn't start with a capital letter.